src/components/Navigation.tsx

```diff
@@ -1,6 +1,9 @@
 import { SUPABASE } from '../helpers/supabaseClient';
 import { Link } from 'react-router-dom';
 import { User, Session } from '../types';
+import { useState, useEffect } from 'react';
+import DarkModeIcon from '@mui/icons-material/DarkMode';
+import LightModeIcon from '@mui/icons-material/LightMode';
```

Comment:
It should be noted that we can use [the babel plugin](https://mui.com/material-ui/guides/minimizing-bundle-size/#option-two-use-a-babel-plugin) to help minimize bundle sizes and use a single line import.

Reply:
We should definitely do this. Created #139